ruzzle_bare_minimum.py

```diff
@@ -159,14 +159,15 @@ def get_prefixes():
 
 
 def word_mults_to_int_array():
-    int_word_mults = []
+    """ Converts word_mults to an array of integers representing the word score multiplies. """
```

Comment:
There's a small typo: multiplies -> multipliers. Can I edit this commit by myself or do you have to do it? I'm still new to github and stuff.

Reply:
You can go to the Files Changed tab, go to the file of interest, select the three dots in the upper right corner, and click Edit to edit the file manually.

Reply:
Thanks for all the help on this project! I'd love to give you some recognition by adding you to a contributors list. What's the easiest way to go about this? Should I just add a contributor section at the bottom of the readme?

Reply:
Some people do the readme approach, and some people add a CONTRIBUTORS.md file where they list contributors - mostly just a matter of preference.